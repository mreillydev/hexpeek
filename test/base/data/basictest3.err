#### unrecognized command
hexpeek: unrecognized command. Try 'help'.
#### no numeric input found
hexpeek: malformed number: no numeric input found
hexpeek: malformed number: no numeric input found
hexpeek: malformed number: no numeric input found
hexpeek: malformed number: no numeric input found
hexpeek: malformed number: no numeric input found
#### offset subceeds minimum recognizable file offset
hexpeek: malformed number: offset subceeds minimum recognizable file offset
#### offset exceeds maximum recognizable file offset
hexpeek: malformed number: offset exceeds maximum recognizable file offset
#### bad negative file offset
hexpeek: malformed number: bad negative file offset
#### unexpected data after number
hexpeek: malformed number: unexpected data after number
#### negative input not valid
hexpeek: malformed number: negative input not valid
#### excessively large input
hexpeek: malformed number: excessively large input
#### line width may not exceed 0x10000 octets
hexpeek: line width may not exceed 0x10000 octets
#### group width may not exceed 0x10000 octets
hexpeek: group width may not exceed 0x10000 octets
#### duplicate '+'
hexpeek: malformed command: duplicate '+'
hexpeek: malformed command: duplicate '+'
#### invalid file number
hexpeek: malformed command: invalid file number
hexpeek: malformed command: invalid file number
hexpeek: malformed command: invalid file number
hexpeek: malformed command: invalid file number
hexpeek: malformed command: invalid file number
hexpeek: malformed command: invalid file number
hexpeek: malformed command: invalid file number
hexpeek: malformed command: invalid file number
#### invalid file offset after '@'
hexpeek: malformed command: invalid file offset after '@'
hexpeek: malformed command: invalid file offset after '@'
hexpeek: malformed command: invalid file offset after '@'
hexpeek: malformed command: invalid file offset after '@'
hexpeek: malformed command: invalid file offset after '@'
hexpeek: malformed command: invalid file offset after '@'
#### negative length not valid
hexpeek: malformed number: negative length not valid
#### duplicate filezone input
hexpeek: malformed command: duplicate filezone input
hexpeek: malformed command: duplicate filezone input
hexpeek: malformed command: duplicate filezone input
hexpeek: malformed command: duplicate filezone input
#### unexpected text after filezone input
hexpeek: malformed command: unexpected text after filezone input
hexpeek: malformed command: unexpected text after filezone input
hexpeek: malformed command: unexpected text after filezone input
hexpeek: malformed command: unexpected text after filezone input
hexpeek: malformed command: unexpected text after filezone input
hexpeek: malformed command: unexpected text after filezone input
hexpeek: malformed command: unexpected text after filezone input
hexpeek: malformed command: unexpected text after filezone input
hexpeek: malformed command: unexpected text after filezone input
#### invalid subcommand after Hexoff
hexpeek: malformed command: invalid subcommand after Hexoff
#### invalid subcommand with '+'
hexpeek: malformed command: invalid subcommand with '+'
#### space required before optional argument
hexpeek: malformed command: space required before optional argument
#### space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
hexpeek: malformed command: space required before mandatory argument
#### trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
hexpeek: malformed command: trailing text
#### empty argument
hexpeek: malformed command: empty argument
hexpeek: malformed command: empty argument
hexpeek: malformed command: empty argument
#### octets not fully specified
hexpeek: malformed input: octets not fully specified
hexpeek: malformed input: octets not fully specified
hexpeek: malformed input: octets not fully specified
hexpeek: malformed input: octets not fully specified
#### unrecognized character
hexpeek: malformed input: unrecognized character 'Z'
hexpeek: malformed input: unrecognized character 'Z'
hexpeek: malformed input: unrecognized character 'Z'
hexpeek: malformed input: unrecognized character 'Z'
hexpeek: malformed input: unrecognized character 'g'
hexpeek: malformed input: unrecognized character 'g'
hexpeek: malformed input: unrecognized character 'g'
hexpeek: malformed input: unrecognized character 'g'
hexpeek: malformed input: unrecognized character 'g'
hexpeek: malformed input: unrecognized character 'g'
#### unexpected text after filezone argument
hexpeek: malformed command: unexpected text after filezone argument
hexpeek: malformed command: unexpected text after filezone argument
#### cannot read beyond file length
hexpeek: malformed command: cannot read beyond file length
hexpeek: malformed command: cannot read beyond file length
#### octets not fully specified
hexpeek: malformed input: octets not fully specified
hexpeek: malformed input: octets not fully specified
hexpeek: malformed input: octets not fully specified
hexpeek: malformed input: octets not fully specified
hexpeek: malformed input: octets not fully specified
#### unrecognized character
hexpeek: malformed input: unrecognized character 'l'
hexpeek: malformed input: unrecognized character 'Y'
hexpeek: malformed input: unrecognized character 'Y'
hexpeek: malformed input: unrecognized character 'Y'
hexpeek: malformed input: unrecognized character 'Y'
hexpeek: malformed input: unrecognized character 'L'
hexpeek: malformed input: unrecognized character 'L'
hexpeek: malformed input: unrecognized character 'L'
hexpeek: malformed input: unrecognized character 'L'
#### input length exceeds specified length
hexpeek: malformed command: input length exceeds specified length
#### invalid file number
hexpeek: malformed command: invalid file number
hexpeek: malformed command: invalid file number
hexpeek: malformed command: invalid file number
#### invalid argument to reset
hexpeek: malformed command: invalid argument to reset
hexpeek: malformed command: invalid argument to reset
hexpeek: malformed command: invalid argument to reset
hexpeek: malformed command: invalid argument to reset
#### invalid file number
hexpeek: malformed command: invalid file number
