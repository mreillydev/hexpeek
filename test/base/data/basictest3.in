margin 2
cols 20
# print file out
0:-0p
#### unrecognized command
garbage
#### no numeric input found
,
:
rlen ,
line ?
group z
#### offset subceeds minimum recognizable file offset
-8000000000000000
#### offset exceeds maximum recognizable file offset
8000000000000000
#### bad negative file offset
-2221
#### unexpected data after number
group 1i
#### negative input not valid
group -1
#### excessively large input
group 10000000000000000
#### line width may not exceed 0x10000 octets
line 10001
#### group width may not exceed 0x10000 octets
group 10001
#### duplicate '+'
2++
3 + +
#### invalid file number
$1
$
$ 0
$ 0 @0
$-1
$-1 @0
  $2 @0
$z   @0
#### invalid file offset after '@'
@
@ 3
$0@
@+
@,
@:
#### negative length not valid
1,-1
#### duplicate filezone input
10,20 ,30
10,20 :30
@10:20 ,30
10:20 :30
#### unexpected text after filezone input
$0q
$0  z
10w
10,20,30
@10,20:30
10:20,30
10:20:30
10 30
10 endian
#### invalid subcommand after Hexoff
0 settings
#### invalid subcommand with '+'
1+k
#### space required before optional argument
helpk
#### space required before mandatory argument
rlen
line
group
insert
insert 
replace
replace 
insert20
search
searchAB
#### trailing text
k 4
k i
ki
kIll
killy
deleted
filese
files e
hex upper
hexlo
text ascii e
text ascci
#### empty argument
/
i
r
#### octets not fully specified
/ABC
/ ABC
/A B
/ 0123 4 5
#### unrecognized character
/ZA
/AZ
/ ZA
/ AZ
/g210
/g2 10
/ g210
/21g0
/210g
/21 0g
#### unexpected text after filezone argument
i @1,20.
5 r @3 7
#### cannot read beyond file length
0 r @2221
0 r @2220,2
#### octets not fully specified
i 1
r 123
replace 356 7
insert 1 9
r0 2
#### unrecognized character
10 rlen 4
rY0
r Y0
r0Y
r 0Y
i20ABL502
i20AB5L02
i20AB35L2
i20AB350L
#### input length exceeds specified length
3,2rAABBCC
#### invalid file number
reset $-1
reset $1
reset $01
#### invalid argument to reset
reset $0j
reset $0 1
reset 0
reset a
#### invalid file number
reset $*
# print file out
0:maxp
