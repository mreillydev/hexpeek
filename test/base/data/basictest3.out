At 0 (2220 octets requested, 20 per line, hexadecimal) :
0000: 11223344 55667788 11223344 55667788 11223344 55667788 11223344 55667788
*
2200: 11223344 55667788 11223344 55667788 11223344 55667788 11223344 55667788
At 0 (7fffffffffffffff octets requested, 20 per line, hexadecimal) :
0000: 11223344 55667788 11223344 55667788 11223344 55667788 11223344 55667788
*
2200: 11223344 55667788 11223344 55667788 11223344 55667788 11223344 55667788
